customer-api/src/Controller/StringManipulationController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace App\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+
+class StringManipulationController extends AbstractController
+{
+    /**
+     * @Route("/string-manipulate", name="string_manipulate")
+     */
+    public function index(Request $request): Response
+    {
+        $inputString = $request->query->get('input');
+        $targetString = $request->query->get('target');
+        $result = null;
+
+        if ($inputString !== null && $targetString !== null) {
+            $result = $this->validateStringFromInput($inputString, $targetString) ? 'True' : 'False';
+        }
+
+        return $this->render('string_manipulation/index.html.twig', [
+            'result' => $result,
+        ]);
+    }
+
+
```

Comment:
please give it a more meaningful name 'validateStringFromInput' is very general, also it doesn't show it needs a target to match with in any meaningful way